connect.yaml

```diff
@@ -3,7 +3,7 @@ deployAs:
     applicationType: assets
   - name: processor
     applicationType: service
-    endpoint: /
+    endpoint: /notifications
```

Comment:
@prateek-ct I guess this will cause to return the deployment url in the form `<url>/notifications`. This will cause checkout to breack as we use this url to then append the path, so at the moment this is breaking change for checkout.

Also I wanted to understand why this endpoint needs to be `/notifications` and it cannot be the root path.

Reply:
Correct the deployment application url will change. Currently users are blind on the notifications url until guided to add. 
All the URLs (payment-methods , status etc) are generated programmatically?

Reply:
Going to defer the PR